test/test_entrypoint.py

```diff
@@ -27,8 +33,39 @@ def test_trivial_argparse():
 
 def test_mocked_cc_pipeline(tmpdir):
     args = parse_args([str(tmpdir), '-a', 'aarch64', '-o', 'ubuntu'])
-    with patch('ros_cross_compile.ros_cross_compile.DockerClient', Mock()) as docker_mock:
+    with patch('ros_cross_compile.ros_cross_compile.DockerClient', Mock()) as docker_mock,\
+            patch('ros_cross_compile.ros_cross_compile.check_install_script_exists') as sc_mock:
```

Comment:
nitpick: align the lines
```suggestion
        patch('ros_cross_compile.ros_cross_compile.check_install_script_exists') as sc_mock:
```